(node:3733888) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-42203',
  containerIp: '10.0.3.164',
  mitmPort: 42203,
  autoAccessFixed: 1
}
2024-11-08 08:18:46.741 - [Info] MITM Version: 2
2024-11-08 08:18:46.744 - [Info] Auto Access Enabled: true
2024-11-08 08:18:46.745 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-08 08:18:46.759 - [Info] SSH man-in-the-middle server for 10.0.3.164 listening on 10.0.3.1:42203
2024-11-08 08:19:27.360 - [Debug] [Connection] Attacker connected: 185.17.229.65 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 08:19:28.113 - [Debug] [Auth] Attacker 185.17.229.65 trying to authenticate with 'password'
2024-11-08 08:19:28.114 - [Debug] [Auto Access] Attacker: 185.17.229.65, Threshold: 1, Attempts: 1
2024-11-08 08:19:28.115 - [Debug] [Auto Access] Compromising the honeypot
2024-11-08 08:19:28.115 - [Debug] [Auto Access] Adding the following credentials: 'merli:merli'
2024-11-08 08:19:28.699 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-08 08:19:28.701 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-08 08:19:28.702 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.164
2024-11-08 08:19:28.783 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-08 08:19:29.325 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-08 08:19:30.106 - [Debug] [Connection] Attacker connected: 185.17.229.65 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 08:19:30.861 - [Debug] [Auth] Attacker 185.17.229.65 trying to authenticate with 'password'
2024-11-08 08:19:30.861 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-08 08:19:30.862 - [Debug] [Auth] Attacker: 185.17.229.65 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 08:19:31.133 - [Debug] [Connection] Attacker closed the connection
2024-11-08 08:19:31.243 - [Debug] [Connection] Attacker connected: 185.17.229.65 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 08:19:31.971 - [Debug] [Auth] Attacker 185.17.229.65 trying to authenticate with 'password'
2024-11-08 08:19:31.972 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-08 08:19:31.972 - [Debug] [Auth] Attacker: 185.17.229.65 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 08:19:32.239 - [Debug] [Connection] Attacker closed the connection
2024-11-08 08:19:32.482 - [Debug] [Connection] Attacker closed connection
2024-11-08 08:19:32.490 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-08 08:19:32.491 - [Debug] [LXC] Container's OpenSSH server closed connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
